### DISCUSSION TITLE
Design talks

### DISCUSSION THREAD

**Discussion by KarenDouglas** [remix-austin/remixaustin-com#27] [Ideas]
First of all, thanks for inviting me to contribute to RemixAustin.com. I haven't been a part of building a site this early in its production, so please bear with me as I am learning!

**One thing I noticed is the navbar doesn't have Discord as an icon.** It is my opinion, that it would be better suited up there, instead of at the bottom of the screen.

![navbar after](https://user-images.githubusercontent.com/79128405/204642484-a696e7e0-93ea-4836-9fed-94f5a739544f.png)

Above I found an icon that would work. 
**I can push my icon contribution if this is ok?**

I would also like to brainstorm giving the "Want to Present" card more room and content on this page. In addition,  a post on "Why you should present",  would be a great post to push more people to present. Let me know what you think.

**Comment by brookslybrand** [comment]
C.C. @sethdavis512 to see if you have any opinions.

I'm for moving the Discord to the socials up above, I think Seth was mostly trying to even out the cards at the bottom. Definitely open to see what you have in mind for the "Want to Present" section.

Also, I love the idea of a post for "Why you should present". That would be a great page to build out and probably means we should spin up some tickets for handling markdown support/a place for these sorts of things

**Reply by sethdavis512** [reply]
@KarenDouglas - Are you looking to add the Discord CTA to the nav and remove the card below? Or just adding the Discord CTA? Either way is fine with me - I'm just having fun out here 😄 

> I love the idea of a post for "Why you should present"  

I'll +1 that @brookslybrand

**Comment by KarenDouglas** [comment]
@sethdavis512 Maybe we should just add the icon to the Nav until other content can fill the second card... like the "why you should present" or "Why you should attend the meetup" type of content.
BTW I hope it didn't come off as overly critical of your cards! I really like the card and feel that they will be great for featured blog posts.

**Reply by sethdavis512** [reply]
> BTW I hope it didn't come off as overly critical of your cards!

No you did not come across as critical at all. I enjoy feedback so please feel free to say whatever comes to mind.

I'll get that Discord link in soon 😄

**Reply by KarenDouglas** [reply]
@sethdavis512

> No you did not come across as critical at all. I enjoy feedback so please feel free to say whatever comes to mind.

ok great! phew!

I was planning to implement my suggestion myself, as I want to get something on the board :)! I MUST CONTRIBUTE! lol, Next time I will call dibs on these discussions to avoid overlap.

**Reply by sethdavis512** [reply]
Ah crap - I'm so sorry about that! I did not mean to step on any toes. I will try to be more aware next time.

I don't mind taking down the PR if you'd like to tackle this one. Let me know. Again sorry about the misstep.

**Reply by KarenDouglas** [reply]
> Ah crap - I'm so sorry about that! I did not mean to step on any toes. I will try to be more aware next time.

OH no problem, I could have been clearer in the discussion. I reread and realized that I didn't outright say "I will do this". I just made an "issue" out of it and assigned myself to it, without letting anyone know my plans in the discussion. So I can see how this happened. I am fairly new to this type of collaboration, so I am learning. I will be more vocal moving forward :D.

> I don't mind taking down the PR if you'd like to tackle this one. Let me know. Again sorry about the misstep.


No, please don't. I will just step up my communication skills! Thank you for even suggesting that, though!

**Reply by sethdavis512** [reply]
> I am fairly new to this type of collaboration, so I am learning

Same here, still finding my rhythm...You're welcome - glad we sorted things out 😄

**Reply by brookslybrand** [reply]
Thanks so much @KarenDouglas for contributing ideas and wanting to contribute code 🙂 

I just added #40 and #41 and assigned them to you if you want to get your name on the board 🤘🏼

Also, while I'm here, thanks @sethdavis512  and @grantnorwood for crushing it with contributions! Definitely was getting some FOMO seeing all the notifications coming in my inbox and not having time to help. Landing site is already looking great

**Comment by grantnorwood** [comment]
What is the correct link for our Discord channel?  Is it public, or do we need to send invites?

The current link points here:  https://discord.com/channels/770287896669978684/953371044108972112

However, it seems to be broken ...
<img width="530" alt="image" src="https://user-images.githubusercontent.com/707463/205184970-c5ce29ae-86d9-445b-8d9f-288f68a92318.png">

**Reply by sethdavis512** [reply]
Hrm...I'm not seeing the same screen that you're getting

https://user-images.githubusercontent.com/3208705/205201106-ee09b8fc-0e85-465e-81ad-5caaacbfa158.mov

**Reply by grantnorwood** [reply]
Seems like a permissions or invite-related issue.  I definitely don't see all of that.  Is it a public channel?

**Reply by sethdavis512** [reply]
Yeah it could be invite related...did some digging - found this button:

<img width="276" alt="image" src="https://user-images.githubusercontent.com/3208705/205206537-6e6aa7e7-498c-465f-a27e-e317306e52a2.png">

<img width="429" alt="image" src="https://user-images.githubusercontent.com/3208705/205206682-bce4d2bc-905a-437a-a883-a0504f99d87e.png">

Mind trying this one? https://discord.gg/8tuARStXkj

**Reply by sethdavis512** [reply]
Hrrmmm, not a fan of how it says "Seth Davis invited you blah blah"... @brookslybrand - you want your name there instead? 😛 

![remix-discord-invite](https://user-images.githubusercontent.com/3208705/205208182-24e93867-f154-47dd-a56e-f055c9373706.jpg)

**Reply by grantnorwood** [reply]
Thanks, that invite link worked!

**Comment by brookslybrand** [comment]
Calling bankruptcy on all discussions and issues today. Going to start revamping the direction of this project, so stay tuned 🚀
